docs/pbc/all/payment-service-providers/payone/integrate-payone.md

```diff
@@ -40,5 +40,30 @@ To integrate Payone, follow these steps.
 If the app was connected successfully, a corresponding message appears, and the app status changes to **Connected**. The payment methods you've selected in step 8, appear in **Administration&nbsp;<span aria-label="and then">></span>  Payment methods**:
 ![payone-credit-card](https://spryker.s3.eu-central-1.amazonaws.com/docs/aop/user/apps/payone/payone-credit-card.png).
 
+## 3. Whitelist Payone Domain
```

Comment:
We use allowlist/denylist instead of whitelist/blacklist. Also, these can be used as nouns only. I rephrased the title accordingly.
```suggestion
## 3. Add Payone domain to your allowlist
```

Reply:
The problem is that here we reference a technical term inside the code; see the code example below. I am fully on to call it allowlist instead of whitelist but please double-check the understandability of the doc with both elements being displayed.

Reply:
I think as long as the technical example is clear, there is no problem with calling it allowlist/denylist as the meaning of both are the same.